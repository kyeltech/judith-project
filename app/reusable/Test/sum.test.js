//import liraries
import React from 'react';
import renderer from 'react-test-renderer';
import Sum from './sum';

test('render wetin them tell you', () => {
  const tree = renderer.create(<Sum />).toJSON();
  expect(tree).toMatchSnapshot();
});
