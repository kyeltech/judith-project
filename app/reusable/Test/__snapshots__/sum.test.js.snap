// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render wetin them tell you 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "backgroundColor": "#F5FCFF",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    style={
      {
        "fontSize": 20,
        "margin": 10,
        "textAlign": "center",
      }
    }
  >
    Welcome to React Native!
  </Text>
  <Text
    style={
      {
        "color": "#333333",
        "marginBottom": 5,
        "textAlign": "center",
      }
    }
  >
    This is a React Native snapshot test.
  </Text>
</View>
`;
